mount/default.conf

```diff
@@ -53,20 +68,34 @@ CVMFS_KEYS_DIR=/etc/cvmfs/keys
 # If CernVM-FS switches to a backup proxy group, reset after X seconds
 # Unset or set to 0 to disable
 CVMFS_PROXY_RESET_AFTER=300
+
 # Same as CVMFS_PROXY_RESET_AFTER for hosts
 CVMFS_HOST_RESET_AFTER=1800
+
+# Retries for failed HTTP requests
 CVMFS_MAX_RETRIES=1
+
+# For HTTP request retries, start with a backoff of 2s
 CVMFS_BACKOFF_INIT=2
+
+# Cut-off of the exponential backoff for HTTP retries at 10s
 CVMFS_BACKOFF_MAX=10
+
+# Transmit the logical path name when requesting a content-addressed object
 CVMFS_SEND_INFO_HEADER=no
+
+# Depending on the stratum 1 support, this options is usually turned on
+# by the domain-specific configuration.
 CVMFS_USE_GEOAPI=no
+
 # If the transfer rate falls below the given bytes/s for longer than
 # CVMFS_TIMEOUT[_DIRECT], treat the connection like a timeout.
 CVMFS_LOW_SPEED_LIMIT=1024
 
-# CA and CRL files used to verify repository signatures
-# EXPERIMENTAL!
-# CVMFS_TRUSTED_CERTS=/etc/grid-security/certificates
+# Add the next line to the default.local file in order to use the OpenHTC
```

Comment:
This too I think doesn't belong in /etc/cvmfs/default.local, because its behavior only applies to the cvmfs-config-default configuration.  I intend to include the same control in cvmfs-config-osg & cvmfs-config-egi, but it will sometimes be enabled by default.